api-tests/features/p2-coi-delete-identity.feature

```diff
@@ -0,0 +1,73 @@
+@Build
+Feature: Delete identity
+
+  Scenario: Fast Follow COI - Delete
+    # Initial journey
+    Given TICF CRI will respond with default parameters and
+      | responseDelay | 0         |
+    When I start a new 'medium-confidence' journey with feature set 'updateDetailsAccountDeletion,ticfCriBeta'
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'page-ipv-success' page response
```

Comment:
Chris has set up a convenient step to speed up these 'setup' journeys. Should match the other COI tests:
```
        Given the subject already has the following credentials
            | CRI     | scenario                     |
            | dcmaw   | kenneth-driving-permit-valid |
            | address | kenneth-current              |
            | fraud   | kenneth-score-2              |
```
If it's the same in both tests, you could also put it in a `Background` to avoid duplication :)